python/tenzir/tools/python_operator_executor.py

```diff
@@ -341,48 +383,55 @@ def main() -> int:
     """
     # The parent uses `codepipe` to transfer the user code to be executed
     # into this program.
-    codepipe = int(sys.argv[1])
-    code = os.read(codepipe, 128 * 1024)
-
+    codepipe = os.fdopen(int(sys.argv[1]), "r")
+    code = codepipe.read()
+    codepipe.close()
+    source_name = "<inline>"
+    lines = code.splitlines(keepends=True)
+    linecache.cache[source_name] = len(code), None, lines, source_name
 
     # When the parent detects an error, it attempts to read the contents
     # of `errpipe` and aborts the pipeline with them as the error.
-    errpipe = int(sys.argv[2])
-
-    # The parent uses stdin and stdout to transfer the arrow record batches
-    # back and forth.
-    istream = pa.input_stream(sys.stdin.buffer)
-    ostream = pa.output_stream(sys.stdout.buffer)
-
-    try:
-        compiled_code = compile(code, '<string>', 'exec')
-        while True:
-            reader = pa.ipc.RecordBatchStreamReader(istream)
-            batch_in = reader.read_next_batch()
-            # The writer writes an invalid record batch as end-of-stream
-            # marker; we have to read it now to remove it from the pipe
-            # buffer.
-            try:
-                reader.read_next_batch()
-            except StopIteration:
-                pass
-            batch_out = execute_user_code(batch_in, compiled_code)
-            writer = pa.ipc.RecordBatchStreamWriter(ostream, batch_out.schema)
-            writer.write_batch(batch_out)
-            writer.close()
-            sys.stdout.flush()
-    except pa.lib.ArrowInvalid as ae:
-        # The reader throws `ArrowInvalid` when the input is closed
-        # by the parent process.
-        pass
-    except Exception as e:
-        message = str(e).encode("utf-8")
-        # Ensure we will never block while trying to write the error message
-        # by truncating to a size that's smaller than the OS pipe buffer.
-        MAX_MSG_SIZE = 4 * 1024
-        if len(message) > MAX_MSG_SIZE:
-            message = message[:MAX_MSG_SIZE]
-        os.write(errpipe, message)
-        os.close(errpipe)
-        return 1
+    errfd = int(sys.argv[2])
+    errpipe_bufsize = fcntl.fcntl(errfd, fcntl.F_GETPIPE_SZ)
```

Comment:
This is only available on Linux.

Reply:
This should work as a fallback:

```diff
diff --git a/python/tenzir/tools/python_operator_executor.py b/python/tenzir/tools/python_operator_executor.py
index 3c1054b0e9..96549804ca 100644
--- a/python/tenzir/tools/python_operator_executor.py
+++ b/python/tenzir/tools/python_operator_executor.py
@@ -393,7 +393,13 @@ def main() -> int:
     # When the parent detects an error, it attempts to read the contents
     # of `errpipe` and aborts the pipeline with them as the error.
     errfd = int(sys.argv[2])
-    errpipe_bufsize = fcntl.fcntl(errfd, fcntl.F_GETPIPE_SZ)
+    try:
+        errpipe_bufsize = fcntl.fcntl(errfd, fcntl.F_GETPIPE_SZ)
+    except AttributeError:
+        # F_GETPIPE_SZ is not available on macOS, so we fall back to getting
+        # fstate to get the file size limit on macOS, or otherwise fall back to
+        # 64 KiB.
+        errpipe_bufsize = os.fstat(errfd).st_size or 65536
     with writeLimited(os.fdopen(errfd, "a"), errpipe_bufsize) as errpipe:
 
         # The parent uses stdin and stdout to transfer the arrow record batches
```

However, now I run into problems with the global AWS S3 SDK state when using the `python` operator:

```
/tmp/apache-arrow-20240611-62116-oihqnj/apache-arrow-16.1.0/cpp/src/arrow/filesystem/s3fs.cc:3093:  arrow::fs::FinalizeS3 was not called even though S3 was initialized.  This could lead to a segmentation fault at exit
libc++abi: terminating
tenzir-4.17.3+gf156ab8b4b-dirty: Error: signal 6 (Abort trap: 6)
0x704400010cda8518: (fatal_handler+0x50)
0xb16c800194c47584: (_sigtramp+0x38)
0xe632000194c16c20: (pthread_kill+0x120)
0x4348800194b23a30: (abort+0xb4)
0x4245000194bcdd08: (__cxxabiv1::__aligned_malloc_with_fallback(unsigned long)+0x0)
0xa070000194bbde98: (demangling_terminate_handler()+0x34)
0x8616000194bcd0cc: (std::__terminate(void (*)())+0x10)
0x2b0f000194bcd028: (std::terminate()+0x24)
0x4817000194b4ea84: (std::__1::thread::join()+0x0)
0xa14b80010846f00c: (std::__1::vector<std::__1::thread, std::__1::allocator<std::__1::thread>>::__destroy_vector::operator()[abi:ue170006]()+0x38)
0x4f1a80010846e1fc: (spdlog::details::thread_pool::~thread_pool()+0x98)
0x2438000108459c64: (std::__1::shared_ptr<spdlog::details::thread_pool>::~shared_ptr[abi:ue170006]()+0x38)
0xf1c0001084534f0: (spdlog::details::registry::~registry()+0x2c)
0xe2f000194ad42e8: (__cxa_finalize_ranges+0x1dc)
0x8b46000194ad4070: (exit+0x2c)
0x173f00010462a3d8: (bool boost::process::detail::posix::wait_until<std::__1::chrono::steady_clock, std::__1::chrono::duration<long long, std::__1::ratio<1l, 1000000000l>>>(boost::process::detail::posix::child_handle const&, int&, std::__1::chrono::time_point<std::__1::chrono::steady_clock, std::__1::chrono::duration<long long, std::__1::ratio<1l, 1000000000l>>> const&, std::__1::error_code&)+0x28c)
0x1e70000104617a28: (bool boost::process::child::wait_for<long long, std::__1::ratio<1l, 1l>>(std::__1::chrono::duration<long long, std::__1::ratio<1l, 1l>> const&)+0x70)
0xa80180010462f708: (tenzir::plugins::python::(anonymous namespace)::python_operator::execute(tenzir::generator<tenzir::table_slice>, tenzir::operator_control_plane&) const (.resume)+0x2b94)
0x320a00010cfa588c: (caf::detail::default_behavior_impl<std::__1::tuple<caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::internal, tenzir::atom::run), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>&), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::pause), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::resume), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::diagnostic&), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::push, tenzir::table_slice&), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>&), caf::typed_behavior<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)> tenzir::(anonymous namespace)::exec_node<tenzir::table_slice, tenzir::table_slice>(caf::stateful_actor<tenzir::(anonymous namespace)::exec_node_state<tenzir::table_slice, tenzir::table_slice>, caf::typed_event_based_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>>*, std::__1::unique_ptr<tenzir::operator_base, std::__1::default_delete<tenzir::operator_base>>, caf::typed_actor<caf::result<tenzir::rest_response> (tenzir::atom::proxy, tenzir::http_request_description), caf::result<void> (tenzir::atom::internal, caf::spawn_atom, tenzir::atom::plugin), caf::result<caf::actor> (caf::spawn_atom, tenzir::invocation), caf::result<caf::ok_atom> (caf::put_atom, caf::actor, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::type, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<caf::actor> (caf::get_atom, tenzir::atom::label, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>), caf::result<std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>> (caf::get_atom, tenzir::atom::label, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>>), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (caf::get_atom, tenzir::atom::version), caf::result<tenzir::detail::vector_map<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, tenzir::data>>, tenzir::detail::stable_map_policy>> (tenzir::atom::config), caf::result<caf::typed_actor<caf::result<void> (tenzir::atom::internal, tenzir::atom::run), caf::result<void> (tenzir::atom::start, std::__1::vector<caf::actor, std::__1::allocator<caf::actor>>), caf::result<void> (tenzir::atom::pause), caf::result<void> (tenzir::atom::resume), caf::result<void> (tenzir::diagnostic), caf::result<void> (tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>, unsigned long long), caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>> (caf::spawn_atom, tenzir::operator_box, tenzir::tag_variant<void, tenzir::table_slice, caf::intrusive_ptr<tenzir::chunk>>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int)>, caf::typed_actor<caf::result<void> (tenzir::diagnostic)>, caf::typed_actor<caf::result<void> (tenzir::metric)>, int, bool)::'lambda'(tenzir::atom::pull, caf::typed_actor<caf::result<void> (tenzir::atom::push, tenzir::table_slice), caf::result<void> (tenzir::atom::push, caf::intrusive_ptr<tenzir::chunk>)>&, unsigned long long)>, caf::detail::dummy_timeout_definition>::invoke(caf::detail::invoke_result_visitor&, caf::message&)+0x1148)
0x447980010b1e199c: (caf::scheduled_actor::consume(caf::mailbox_element&)+0x2dc)
0xe20980010b1e20c8: (caf::scheduled_actor::reactivate(caf::mailbox_element&)+0x20)
0x916a80010b1e6128: (caf::scheduled_actor::resume(caf::execution_unit*, unsigned long)::$_3::operator()(caf::mailbox_element&) const+0x94)
0xa7800010b1de00c: (caf::scheduled_actor::resume(caf::execution_unit*, unsigned long)+0x450)
0xdd1380010b0fbea0: (caf::scheduler::worker<caf::policy::work_stealing>::run()+0x68)
0x191380010b0fbdbc: (void std::__1::__thread_execute[abi:ue170006]<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct>>, std::__1::thread caf::actor_system::launch_thread<caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'()>(char const*, caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'())::'lambda'(caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'()), caf::intrusive_ptr<caf::ref_counted>, 2ul>(std::__1::tuple<caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'(), std::__1::thread caf::actor_system::launch_thread<caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'()>(char const*, caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'())::'lambda'(caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'()), caf::intrusive_ptr<caf::ref_counted>>&, std::__1::__tuple_indices<2ul>)+0x6c)
0xfc5880010b0fbc78: (void* std::__1::__thread_proxy[abi:ue170006]<std::__1::tuple<std::__1::unique_ptr<std::__1::__thread_struct, std::__1::default_delete<std::__1::__thread_struct>>, std::__1::thread caf::actor_system::launch_thread<caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'()>(char const*, caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'())::'lambda'(caf::scheduler::worker<caf::policy::work_stealing>::start()::'lambda'()), caf::intrusive_ptr<caf::ref_counted>>>(void*)+0x34)
0xba4d000194c16f94: (_pthread_start+0x88)
```